SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h

```diff
@@ -70,169 +70,142 @@ class SPATIALGDK_API USpatialGDKSettings : public UObject
-	 * (If you set the value to ` 0`, the SpatialOS Runtime replicates every Actor per tick; this forms a large SpatialOS  world, affecting
-	 * the performance of both game clients and server-worker instances.) You can use the `stat Spatial` flag when you run project builds to
-	 * find the number of calls to `ReplicateActor`, and then use this number for reference.
+	 * Warning: default value 0 means that every Actor replicates. Ignored when using Replication Graph.
 	 */
-	UPROPERTY(EditAnywhere, config, Category = "Replication", meta = (DisplayName = "Maximum Actors replicated per tick"))
+	UPROPERTY(EditAnywhere, config, Category = "Replication", meta = (DisplayName = "Maximum Actors Replicated Per Tick"))
 	uint32 ActorReplicationRateLimit;
 
 	/**
-	 * Specifies the maximum number of entities created by the SpatialOS Runtime per tick. Not respected when using the Replication Graph.
-	 * (The SpatialOS Runtime handles entity creation separately from Actor replication to ensure it can handle entity creation requests
-	 * under load.) Note: if you set the value to 0, there is no limit to the number of entities created per tick. However, too many
-	 * entities created at the same time might overload the SpatialOS Runtime, which can negatively affect your game. Default: `0` per tick
-	 * (no limit)
+	 * Warning: default value 0 means there is no upper limit. Ignored when using Replication Graph.
 	 */
-	UPROPERTY(EditAnywhere, config, Category = "Replication", meta = (DisplayName = "Maximum entities created per tick"))
+	UPROPERTY(EditAnywhere, config, Category = "Replication", meta = (DisplayName = "Maximum Entities Created Per Tick"))
 	uint32 EntityCreationRateLimit;
 
 	/**
-	 * When enabled, only entities which are in the net relevancy range of player controllers will be replicated to SpatialOS. Not respected
-	 * when using the Replication Graph. This should only be used in single server configurations. The state of the world in the inspector
-	 * will no longer be up to date.
+	 * Relevancy as defined by Unreal. Only use for single server configurations. Ignored when using Replication Graph.
 	 */
 	UPROPERTY(EditAnywhere, config, Category = "Replication", meta = (DisplayName = "Only Replicate Net Relevant Actors"))
 	bool bUseIsActorRelevantForConnection;
 
 	/**
-	 * Specifies the rate, in number of times per second, at which server-worker instance updates are sent to and received from the
-	 * SpatialOS Runtime. Default:1000/s
+	 * Between server-workers and the SpatialOS Runtime.
 	 */
 	UPROPERTY(EditAnywhere, config, Category = "Replication", meta = (DisplayName = "SpatialOS Network Update Rate"))
 	float OpsUpdateRate;
 
 	/**
-	 * Maximum NetCullDistanceSquared value used in Spatial networking. Not respected when using the Replication Graph.
-	 * Set to 0.0 to disable. This is temporary and will be removed when the runtime issue is resolved.
+	 * The upper limit for Net Cull Distance Squared. Applies to all Actors. 
```

Comment:
* The upper limit for every Actor's `NetCullDistanceSquared` property (see [GDK documentation](https://documentation.improbable.io/gdk-for-unreal/docs/game-client-interest-management#section-net-cull-distance-squared)).  Set to `0.0` to disable.  Note: Ignored when using Replication Graph. 

//Suggested a link to the docs (if this is possible) as this is a SpatialOS special. Also suggested a way to remove an extra sentence.